core/util_test.go

```diff
@@ -34,6 +34,12 @@ func TestNewToken(t *testing.T) {
 	return
 }
 
+func TestLooksLikeAToken(t *testing.T) {
+	test.Assert(t, !LooksLikeAToken("R-ULy7MrV3tUUjO9v5ym2srK3dGGCwlxbVyKBdwLOS"), "Accepted short token")
+	test.Assert(t, !LooksLikeAToken("R-ULy7MrV3tUUjO9v5ym2srK3dGGCwlxbVyKBdwLOS%"), "Accepted invalid token")
+	test.Assert(t, LooksLikeAToken("R-ULy7MrV3tUUjO9v5ym2srK3dGGCwlxbVyKBdwLOSU"), "Rejected valid token")
```

Comment:
This test set should include all of the characters in the set.  Underscore is currently missing.